google/cloud/vision/entity.py

```diff
@@ -52,17 +57,21 @@ def from_api_repr(cls, response):
         :type response: dict
         :param response: Dictionary response from Vision API with entity data.
 
-        :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
+        :rtype: :class:`~google.cloud.vision.entity.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = []
```

Comment:
Is `None` a better default value, or will callers try to iterate over it blindly?